spec/prog/minio/minio_server_nexus_spec.rb

```diff
@@ -11,7 +11,11 @@
       name: "minio-cluster-name",
       admin_user: "minio-admin",
       admin_password: "dummy-password",
-      private_subnet_id: ps.id
+      private_subnet_id: ps.id,
+      root_cert_1: "-----BEGIN CERTIFICATE-----\nMIIB+zCCAaGgAwIBAgIRAIKIt55a5rFgJkNPAeDUHSswCgYIKoZIzj0EAwIwYDEL\nMAkGA1UEBhMCVVMxETAPBgNVBAoMCFViaWNsb3VkMT4wPAYDVQQDDDVtYzFzN2Zu\nZnI0cnQzcWJ4dmd3cDliOHl3ZSBSb290IENlcnRpZmljYXRlIEF1dGhvcml0eTAe\nFw0yNDAxMjUxMDQ4NDRaFw0yOTAxMjMxMDQ4NDRaMGAxCzAJBgNVBAYTAlVTMREw\nDwYDVQQKDAhVYmljbG91ZDE+MDwGA1UEAww1bWMxczdmbmZyNHJ0M3FieHZnd3A5\nYjh5d2UgUm9vdCBDZXJ0aWZpY2F0ZSBBdXRob3JpdHkwWTATBgcqhkjOPQIBBggq\nhkjOPQMBBwNCAATa+IdeQVs/ayP7QUR1Ogofpo5rqPuse5UTshfQ1m8l5VrjGz2N\nbBOlwtLbLAuBQAaAmNgOD7Qa9M2L4pqeDnaZozwwOjAMBgNVHRMEBTADAQH/MAsG\nA1UdDwQEAwIBBjAdBgNVHQ4EFgQU5epQEnBjbawntHPBolPnrLn3m1AwCgYIKoZI\nzj0EAwIDSAAwRQIhAIDWwWqsrKxCrGAcgOsFiAGuou6Ef6owASLLaIDNXBLIAiAG\nKMjm6YRMYdGIS/z6VbrKUkNAU6trF3KY0r0uDiQ26Q==\n-----END CERTIFICATE-----\n",
+      root_cert_key_1: "-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEIMX784b+8awbA6qoHiDOlWxEE1+y7C06JB7QudZb9a3moAoGCCqGSM49\nAwEHoUQDQgAE2viHXkFbP2sj+0FEdToKH6aOa6j7rHuVE7IX0NZvJeVa4xs9jWwT\npcLS2ywLgUAGgJjYDg+0GvTNi+Kang52mQ==\n-----END EC PRIVATE KEY-----\n",
```

Comment:
Can we find a way to not include real looking dummy secrets to the test as they might cause extra work during future auditing/compliance process.